Tests/js/views/assets/ez-trashview-tests.js

```diff
@@ -34,8 +41,11 @@ YUI.add('ez-trashview-tests', function (Y) {
                 contentType: this.contentTypeMock
             }];
 
+            this.barMock = new RenderedView();
```

Comment:
it's not really a mock, you'd better call it `this.barView`. Also, the `RenderedView` instance sets its `rendered` to true but this is not checked anywhere AFAICS.